pepper-apis/docs/specification/deploy/spec/pepper.yaml

```diff
@@ -0,0 +1 @@
+../../build/pepper.yaml
```

Comment:
Trying to give gcloud app deploy the least number of files to consider.
gcloud app deploy only needs 4 files:
index.html : the main webage
pepper.json and pepper.yaml: the built configurations. pepper.yaml is not really used or visible, but can be downloaded if you know the url.
app.yaml: the GAE deployment configuration file.
pepper.json and pepper.yaml are the only files we care about in our build.
These are soft-links from the deploy directory to the build directory.